Ghidra/Framework/SoftwareModeling/src/main/java/ghidra/program/model/pcode/HighFunction.java

```diff
@@ -64,6 +66,8 @@ public HighFunction(Function function, Language language, CompilerSpec compilerS
 		this.language = language;
 		this.compilerSpec = compilerSpec;
 		AddressSpace stackSpace = function.getProgram().getAddressFactory().getStackSpace();
+		entryPoint = function.getEntryPoint();
+		entryAddrSpace = entryPoint.getAddressSpace();
```

Comment:
Feels like a dangerous micro-optimization, if any other function type even implements a SetAddressSpace or SetEtrypoint() function.

`Function` does appear to have a setBody() that takes an address arg. Is this possible?